src/components/RecipesList.js

```diff
@@ -26,6 +28,11 @@ class RecipesList extends React.Component {
     this.setState({ing: ""})
   };
 
+  onDeleteIngredient(index) {
+    this.state.ingredients.splice(index, 1)
+    this.setState({ ingredients: this.state.ingredients});
```

Comment:
Is there no way to converge these two steps into 1? Not a big deal just a thought.

Reply:
line 32 returns the ingredient that has been deleted. not the array without that ingredient.
The need to be separate, we will refactor it later creating a separate function to set the state.